app/routes/missions.tsx

```diff
@@ -82,6 +76,27 @@ export default function () {
               <NavLink key="sksn" to="sksn">
                 Super-Kamiokande
               </NavLink>,
+              useFeature('SVOM') && (
+                <NavLink key="svom" to="svom">
+                  SVOM
+                </NavLink>
+              ),
+              <>
+                <NavLink key="archival" to="archival">
+                  Archival
+                </NavLink>
+                <SideNavSub
+                  base="archival"
+                  items={[
+                    <NavLink key="agile" to="agile">
+                      AGILE
+                    </NavLink>,
+                    <NavLink key="burstcube" to="burstcube">
+                      BurstCube
+                    </NavLink>,
+                  ]}
+                />
```

Comment:
For the current page highlighting to work, you'll need to move the archival missions' URL paths from, e.g., `/missions/burstcube` to `/missions/archival/burstcube`. Please do that, and add `/missions/burstcube` as a permanent redirect to `/missions/archival/burstcube`.

Since this move is pretty cut-and-dry but the shortcut links are not (see above), I suggest doing the move and adding the archival mission submenu in its own separate PR.

Reply:
This is separated out into PR #2913.